Dockerfile

```diff
@@ -32,16 +32,7 @@ COPY \
 RUN dart pub get
 
 RUN apt-get update
-RUN apt-get install \
-    libxml2-utils \
-    xsltproc \
-    shellcheck \
-    libxslt1.1
-
-# Install docker
-RUN apt-get install -y curl && \
-    curl -sSL https://get.docker.com | sh
```

Comment:
I'm not sure about this one :thinking: I need to remove both lines, otherwise I'm getting errors and can't rebuild devcontainer. But we are using `curl` to install node:

![image](https://github.com/openstax/ce-styles/assets/40228854/974f0317-53d4-4060-bc3d-63152a86ccf3)

So I don't know why it's working :eyes: